src/auth/roles/index.ts

```diff
@@ -0,0 +1,126 @@
+import { RolesInitialiser } from './types';
+const { lazyPromiseSeries } = require('../../../database');
+import * as knex from 'knex';
+
+const rolesInitialiser: RolesInitialiser = (client) => {
+  return  {
+    add: async({ role, userId, orgId }) => {
```

Comment:
Think I prefer the syntax:
```ts
return {
  async add ({ role, userId, orgId }) => {},
  async remove () => {},
}
```

Also I'd prefer not to abbreviate `organisationId` to `orgId`

Reply:
Shorthand syntax for method definitions is only for functions, rather than arrow functions. I'm not sure if there's any added benefit of functions over arrow functions here. Stylistically I find it slightly more readable using arrow functions as the method name is first.

Reply:
```js
return {
  async add = ({ x }) => {
    // ...
  }
}
```

Reply:
Isn't this public class field syntax, which is currently polyfilled by babel (in classes)?